content/vault/v1.16.x/content/partials/deprecation/ldap-null-bind.mdx

```diff
@@ -0,0 +1,15 @@
+## LDAP Authentication Method - deny_null_bind parameter ((#ldap-null-bind))
+
+| Announced | Expected end of support | Expected removal |
+| :-------: | :---------------------: | :--------------: |
+| NOV 2025  | NOV 2025                | NOV 2025         |
+
+The deny_null_bind parameter in the LDAP authentication method is deprecated.
+
+Vault now always denies login attempts with an empty password, regardless of
+ the `deny_null_bind` parameter's value:
+- If your configuration currently sets `deny_null_bind` = true, no action is required.
+- If your configuration currently sets `deny_null_bind` = false, login attempts 
+that previously succeeded with an empty password will now fail.
+
+The parameter will be fully removed in the next major release.
```

Comment:
```suggestion
Vault now denies all login attempts with an empty password, regardless of how
you set the `deny_null_bind` parameter.

As a result, for deployments where `deny_null_bind` is set to `false`, Vault now
rejects login attempts that previously succeeded with an empty password.

If you currently have `deny_null_bind` set to `true`, you do not need to take
action now. But you will need to remove the parameter when you upgrade to the
next major release.
```
Style correction: avoid possessives, write in active voice

It's worth noting that folks will need to take future action if they choose not to modify their configuration now.